README.md

```diff
@@ -1363,10 +1363,8 @@ expect(Paths.get("/root/.ssh/config")).isWritable()
 ```text
 expected that subject: /root/.ssh/config        (sun.nio.fs.UnixPath <1234789>)
 ◆ is: writable
-    » failure at parent path: /root        (sun.nio.fs.UnixPath <1234789>)
-      » access was denied
-      » the owner is root, the group is root
-      » the permissions are u=rwx g= o=
+    » no file system entry exists at this location
```

Comment:
please revert this and other changes in the readme (this is caused by the fact that you are not using the same OS).